contracts/v2/FiatTokenV2_1.sol

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2018-2020 CENTRE SECZ
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+pragma solidity 0.6.12;
+
+import { FiatTokenV2 } from "./FiatTokenV2.sol";
+
+// solhint-disable func-name-mixedcase
+
+/**
+ * @title FiatToken V2.1
+ * @notice ERC20 Token backed by fiat reserves, version 2.1
+ */
+contract FiatTokenV2_1 is FiatTokenV2 {
+    /**
+     * @notice Initialize v2.1
+     * @param lostAndFound  The address to which the locked funds are sent
+     */
+    function initializeV2_1(address lostAndFound) external {
+        // solhint-disable-next-line reason-string
+        require(_initializedVersion == 1);
+
+        uint256 lockedAmount = balances[address(this)];
+        if (lockedAmount > 0) {
+            _transfer(address(this), lostAndFound, lockedAmount);
+        }
+        blacklisted[address(this)] = true;
```

Comment:
Should we emit a Blacklisted event?

Reply:
I didn't think it was necessary, also was worried that some unsophisticated people may report it as CENTRE blacklisting yet another address.